Streamlit documentation tutorial

daezel opened the pull request:
## Please check the following - and then delete this message:

- If you are creating a new page, does it contain the following value at the top, with x being the position in the folder?
```
---
sidebar_position: x
---
```

- If you are creating a new sub-folder, does it have a `_category_.json` file with the appropriate descriptive information?
- Are all your images / videos named appropriately and in the correct location ("image1.png" e.g. will not be accepted)
- Are all your code blocks properly closed off?
- Does your file have an appropriate and descriptive file name, and main header?
- Ensure all files are referenced correctly (They are case sensitive - if you save "image.png" you cannot reference "Image.PNG")

Failure to include these will result in your request not being reviewed.

SassafrasAU reviewed (changes requested):
Handful of small changes needed:

- Can you add the sidebar_position to the top please, see the PR instructions on the exact usage.
- The URLs with < > around them need to either have that removed, or changed to the typical format of [text](url)

daezel commented:
- Updated raw URLs to Markdown link syntax for better readability and professionalism.
- Added 'sidebar_position' to the Streamlit tutorial section to improve navigation.
